data/vocab/library.ttl

```diff
@@ -0,0 +1,74 @@
+@prefix schema: <http://schema.org/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix bf: <http://id.loc.gov/ontologies/bibframe/> .
+@prefix bflc: <http://bibfra.me/vocab/bflc/> .
+@prefix bflcext: <http://id.loc.gov/ontologies/bflc/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix oa: <http://www.w3.org/ns/oa#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix library: <http://bibfra.me/vocab/marc/> .
+
+
+# BIBFRAME Linked Data Extension aka BUILDE Relation 📖🔍
+
+# This RDF vocabulary encompasses a wide range of classes and properties suitable for describing various
+# types of bibliographic resources, scholarly works, and related entities. BUILDE includes a basic set of 
+# core classes and properties known as the 'bflc' layer described here, as well as several extensions:
+
+#	archive: things found in archives
+#	rare books: things found in rare book libraries
+#	scholar: things found in scholarly research and publishing
+#	library: things found in libraries
+
+# This vocabulary is ideal for librarians, information scientists, data curators, and developers working
+# in academic and research settings. It can be employed in creating digital libraries, scholarly databases,
+# cataloging systems, and other applications requiring structured bibliographic information.
+
+# Key Features:
+# - Compatibility with existing ontologies including Schema.org, SKOS, FOAF, and Dublin Core for enhanced interoperability.
+# - Extension of BIBFRAME from the Library of Congress
+# - Comprehensive coverage of bibliographic and scholarly domains.
+# - Clear, self-explanatory labels and definitions for each class and property.
+# - Hierarchical organization of classes using rdfs:subClassOf for structured representation.
+
+# We encourage users to provide feedback and suggestions to continually improve and update this vocabulary.
+# For versioning, updates, and more detailed documentation, please visit [https://bibfra.me/?].
+
+# ------------------------------
+# Created by: EBSCO Information Systems
+# Version: 1.0
+# Created: August 15, 2024
+# Updated: August 15, 2024
+# ------------------------------
+
+# BUILDE Classes
+
+library:Title a rdfs:Class ;
+    rdfs:label "Title"@en ;
+    rdfs:comment "Title of a resource."@en ;
+    rdfs:subClassOf lite:Resource ;
+    owl:sameAs bf:Title .
+
+library:VariantTitle a rdfs:Class ;
+    rdfs:label "Variant Title"@en ;
+    rdfs:comment "Varying Form of Title."@en ;
+    rdfs:subClassOf library:Title ;
+    owl:sameAs bf:VariantTitle .
+
+library:ParallelTitle a rdfs:Class ;
+    rdfs:label "Parallel Title"@en ;
+    rdfs:comment "Title in another language and/or script."@en ;
+    rdfs:subClassOf library:Title ;
+    owl:sameAs bf:ParallelTitle .
+
+# BUILDE Properties
+library:assigningSource a rdf:Property ;
+    rdfs:label "assigning source"@en ;
+    rdfs:comment "Entity assigning metadata, such as a classification number, name, or identifier."@en ;
```

Comment:
We used "assigningSource" property to represent the entity assigning classification number (refer https://folio-org.atlassian.net/browse/MODLD-381).
However, looking at https://bibfra.me/vocab/marc/assigningSource, now I think if "assigningSource" really mean something else. Want to discuss more.